cmd/juju/cloud/addcredential.go

```diff
@@ -360,8 +360,7 @@ func (c *addCredentialCommand) promptCredentialAttributes(p *interact.Pollster,
 }
 
 func (c *addCredentialCommand) promptFieldValue(
-	p *interact.Pollster, attr jujucloud.NamedCredentialAttr,
-) (string, error) {
+	p *interact.Pollster, ctxt *cmd.Context, attr jujucloud.NamedCredentialAttr) (string, error) {
```

Comment:
So do you need the context? I am not seeing anything that uses it...

Reply:
D'oh , that's a merge conflict fix issue. Removed, not needed. It was used to put output to stdout.

I've added a test too, good point!